### PR TITLE
FAQs

### PR DESCRIPTION
**Pull request by RealOrangeOne**
If a team asks a question, we should probably word it into the docs somehow, or make it more obvious.

If a question doesn't fit well into a section, add the question to this page

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ params:
     disableLanguageSwitchingButton: true
     disableAssetsBusting: true
     editURL: "https://github.com/sourcebots/docs/tree/master/content/"
-    ordersectionsby: "title"
+    ordersectionsby: "weight"
```
Comment (sedders123) [inline config.yml]: Does this fall back to 'title' for all sections with the same weight?

File: content/faqs/_index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: FAQs
+weight: 10
+---
+
+Teams often ask us questions, so we've put the most common ones below, so others can learn from them.
+
+{{% expand "Do these FAQs work?" %}}
```
Comment (sedders123) [inline content/faqs/_index.md]: Is it worth coming up with a couple of 'real' FAQs instead of just dummy data?

File: config.yml (hunk 1 above)
Comment (RealOrangeOne) [reply config.yml]: Yes. It was just a hack to move the FAQs section to the bottom of the list

File: content/faqs/_index.md (hunk 2 above)
Comment (kierdavis) [reply content/faqs/_index.md]: I agree, right now this page looks like a test page (even though we intend to add useful content at a later date)

Comment (RealOrangeOne) [reply content/faqs/_index.md]: Well, that's technically what it is. But i'll try and come up with some generic things to fill it out a bit

**Pull request merged**